style.css

```diff
@@ -0,0 +1,125 @@
+body {
+  font-family: sans-serif;
+}
+
+.parent-container {
+  display: flex;
+  position: absolute;
+  height: 718px;
+  width: 375px;
+}
+
+.image {
+  position: absolute;
+  margin-top: 130.85px;
+  width: 375px;
+  height: 587.15px;
+}
+
+.header-section {
+  display: flex;
+  position: absolute;
+  flex-direction: row;
+  margin-top: 44px;
+  width: 100%;
+  height: 48px;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  width: 100px;
+  height: 20px;
+  margin-left: 24px;
+  position: static;
+  font-size: 18px;
+  line-height: 20px;
+  font-style: normal;
+  text-align: center;
+  align-items: center;
+  color: #6070ff;
+  font-weight: bold;
+  padding-right: 187px;
+}
+
+.hamberger-menu {
+  display: flex;
+  position: absolute;
+  height: 20px;
+  width: 16px;
+  margin-right: 35px;
+}
+
+.hamberger-menu a {
+  color: #6070ff;
+}
+
+.headline-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  position: absolute;
+  width: 375px;
+  height: 640px;
+  margin-top: 92px;
+}
+
+.name {
+  display: flex;
+  position: static;
+  top: 114px;
+  left: 24px;
+  height: 150px;
+  width: 327px;
+  font-size: 40px;
+  color: #344563;
+  font-weight: bold;
+}
+
+.description {
+  display: flex;
+  position: absolute;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  width: 327px;
+  height: 146px;
+  text-align: left;
+  color: #344563;
+  top: 276px;
+  left: 24px;
+}
+
+.contact-message {
+  display: flex;
+  position: static;
+}
```

Comment:
- [ ] Please consider using the correct typography here also 😄
<img width="108" alt="Loom_wDeiMCvAXQ" src="https://user-images.githubusercontent.com/81468675/149381133-399f96d3-8ccc-4868-abc0-c99e1c0752e4.png">